commands/host/phpmyadmin

```diff
@@ -7,9 +7,9 @@
 
 DDEV_PHPMYADMIN_PORT=8036
 DDEV_PHPMYADMIN_HTTPS_PORT=8037
-
+BASE_URL=${DDEV_PRIMARY_URL%:*}
 if [ ${DDEV_PRIMARY_URL%://*} = "http" ] || [ -n "${GITPOD_WORKSPACE_ID:-}" ] || [ "${CODESPACES:-}" = "true" ]; then
-    ddev launch $DDEV_PRIMARY_URL:$DDEV_PHPMYADMIN_PORT
+    ddev launch ${BASE_URL}:$DDEV_PHPMYADMIN_PORT
```

Comment:
I think we don't even need to have a full URL here.
It's enough to open a port with `ddev launch :$DDEV_PHPMYADMIN_PORT`

Reply:
Better!